react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+              <li>Xcode (iOS)</li>
+              <li>CocoaPods (iOS)</li>
+            </ol>
+          </div>
+          <aside class="notes">
+            Let’s talk about how can you install and build your React Native app.
+So, what do we need? Let’s take a closer look on dependencies.
+You will need to install Node, Watchman, the React Native command line interface, a JDK, and Android Studio, Xcode (iOS), CocoaPods (iOS). (And some IDE that supports js, for example Visual Studio Code).
+
+The first three are common for both platform (android and iOS).
+
+NodeJS is a Javascript cross-platform runtime environment. React Native uses Node. js, to build your JavaScript code.
+Watchman is a tool by Facebook for watching changes in the filesystem. It is highly recommended you install it for better performance. React Native uses watchman to detect when you've made code changes and then automatically build and push the update to your device without you needing to manually refresh it.
+//TODO
+React Native command line interface
+
+Now, let’s see what we need for android
+
+JDK (Java Development Kit) is a cross-platformed software development environment. The JDK provides tools, such as the Java compiler, used by IDEs and SDKs for developing Java programs. 
+Android Studio is the official IDE for Android.
+
+And, when we talk about what we need for iOS, it’s 
+Xcode (iOS)  is the IDE for iOS
+And CocoaPods (iOS) witch is a dependency manager for Swift and Objective-C Cocoa projects. React Native uses Cocoapods to add iOS dependencies when you add new modules to your application.
+          </aside>
+        </section>
+
+        <section class="text-and-frame-slide">
+          <h4 class="slide-name">Creating a new application</h4>
+          <div>
+            <p>To create a new project just run:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                npx react-native init yourProject
+              </code>
+            </div>
+
+            <p>[Optional]</p>
+            <p>Using a specific version:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                npx react-native init yourProject --version X.XX.X
+              </code>
+            </div>
+
+            <p>Using a specific template:</p>
+            <div class="frame">
+              <code class="js" data-trim data-line-numbers>
+                npx react-native init yourTSProject --template react-native-template-typescript
+              </code>
+            </div>
+          </div>
+
+          <aside class="notes"></aside>
+        </section>
+
+        <section class="topic-name-slide">
+          <h4>Project structure overview</h4>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">Default structure</h4>
+          <div>
+            <img src="assets/images/default_structure.png" alt="simple" />
```

Comment:
Could you highlight files on you screenshot? or blur files that is not important for this slide?

Reply:
Done